### PR TITLE
HW-52048 Only White Theme Implementation

### PR DESCRIPTION
**Pull request by ymoskovko-epam**
Jira Task [HW-52048](https://jira.site1.hyperwallet.local/browse/HW-51681)

### CODE REVIEW

File: Sources/Extensions/UIColor.swift [hunk 1]
```diff
@@ -40,4 +40,35 @@ extension UIColor {
             alpha: alpha
         )
     }
+
+    func lighter(by percentage: CGFloat = 10.0) -> UIColor {
```
Comment (jsaini-hw) [inline Sources/Extensions/UIColor.swift]: Good approach but I think we do not need it for this ticket. When Willem will create new ticket then maybe we can use this approach.

File: Demo/AppDelegate.swift [hunk 2]
```diff
@@ -31,6 +31,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window?.tintColor = Theme.tintColor
```
Comment (jsaini-hw) [inline Demo/AppDelegate.swift]: move this line after applyTheme as tickMark doesn't show up for white theme if this is set 1st before applying the theme

Comment (ymoskovko-epam) [reply Demo/AppDelegate.swift]: fixed

File: Sources/Extensions/UIColor.swift (hunk 1 above)
Comment (ymoskovko-epam) [reply Sources/Extensions/UIColor.swift]: removed from this task impplementation

File: Sources/Extensions/UIToolbar.swift [hunk 3]
```diff
@@ -30,6 +30,7 @@ extension UIToolbar {
                                          style: .plain,
                                          target: target,
                                          action: action)
+        doneButton.tintColor = Theme.Button.color
```
Comment (jsaini-hw) [inline Sources/Extensions/UIToolbar.swift]: good job

File: Demo/AppDelegate.swift [hunk 4]
```diff
@@ -27,11 +27,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
-        // Set the default tint color
-        window?.tintColor = Theme.Button.color
+        //ThemeManager.applyWhiteTheme()
+        ThemeManager.applyTheme()
+
         // Avoid to display a black area during the view transaction in the UINavigationBar.
         window?.backgroundColor = Theme.ViewController.backgroundColor
-        ThemeManager.applyTheme()
+        // Set the default tint color
+        window?.tintColor = Theme.tintColor
```
Comment (jsaini-hw) [inline Demo/AppDelegate.swift]: This needs to be Theme.Button.color

Comment (ymoskovko-epam) [reply Demo/AppDelegate.swift]: fixed

**Pull request merged**